.github/workflows/reusable-dev-deployment.yml

```diff
@@ -87,6 +87,8 @@ jobs:
             --set caddy.image.repository='docker.io/${{ vars.DOCKER_HUB_USERNAME }}/ecamp3-api-caddy' \
             --set postgresql.dbBackupRestoreImage.repository='docker.io/${{ vars.DOCKER_HUB_USERNAME }}/ecamp3-db-backup-restore' \
             --set termsOfServiceLinkTemplate='https://ecamp3.ch/{lang}/tos' \
+            --set newsLink='https://ecamp3.ch/blog' \
+            --set helpLink='https://ecamp3.ch/faq' \
```

Comment:
If the FAQ page is now a help page, I would expect one more question in the FAQ: "My question is not answered here, where can I ask my question?" which links to the contact form.

Reply:
Done that: https://www.ecamp3.ch/de/faq/